src/history/config.go

```diff
@@ -116,6 +117,43 @@ func (a *ActionInstance) Fields() []interface{} {
 	return fields
 }
 
+type EmbedField struct {
+	Name   string `json:"name"`
+	Value  string `json:"value"`
+	Inline bool   `json:"inline"`
+}
+
+func (a *ActionInstance) EmbedFields(callbackID string) map[string]interface{} {
```

Comment:
Can you please put the callbackID to `discord.PostActionMessage()`? The history package should not worry about callback IDs because it never knew about it and should not.